Okta Managed MCP Server - updates (#6336)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/mcp-server/index.md

The Okta Open Source Model Context Protocol (MCP) server securely connects AI agents and Large Language Models (LLMs) to an Okta org. This abstraction layer enables AI agents and LLMs to interact with Okta admin management APIs using natural language commands.

The Okta Open Source MCP Server translates natural language instructions into structured API calls between LLM clients and an Okta org using Okta's Python SDK v3.4.1. This approach reduces security risks and complexity when connecting autonomous AI agents to IAM systems. The architecture ensures that AI actions remain secure, properly scoped, and fully auditable.

IT admins, developers, and security professionals use the Okta Open Source MCP Server to automate Okta admin tasks through AI-powered interfaces.

Okta offers two ways to deploy an MCP server: the Okta Open Source MCP Server and the Okta Managed MCP Server. The following section introduces each deployment option and helps you choose the one that fits your environment.

## Deployment options: Okta Open Source MCP Server and Okta Managed MCP Server

> To integrate AI assistants securely with your identity infrastructure, the Okta Model Context Protocol (MCP) server can be deployed using two distinct hosting options: Okta Open Source MCP Server and Okta Managed MCP Server.

Reviewer: ```suggestion
To integrate AI assistants securely with your identity infrastructure, you can deploy the Okta Model Context Protocol (MCP) server using two distinct hosting options: Okta Open Source MCP Server and Okta Managed MCP Server.
```

> You can choose the deployment track that matches your technical requirements:

Reviewer: ```suggestion
Choose the deployment method for your technical requirements:
```

> You can choose the deployment track that matches your technical requirements:

Reviewer: Is "track" used in the UI somewhere? I feel like it'll be a translation issue.

> * **Okta Open Source MCP Server**: You run the server software locally on your own infrastructure. This track is best for developers who want to modify the underlying server code. See [Okta Open Source MCP Server documentation](/docs/guides/mcp-server/main/).

Reviewer: ```suggestion
* **Okta Open Source MCP Server**: You run the server software locally on your own infrastructure. This method is best for developers who want to modify the underlying server code. See [Okta Open Source MCP Server documentation](/docs/guides/mcp-server/main/).
```

Reviewer: Who is "you"? Is it an admin or developer?

Author: developer

Author: updated both as "you"

> * **Okta Managed MCP Server**: Okta hosts and manages the server in the cloud. Your chat client streams instructions over secure HTTPS, which removes local software installations and hosting fees. This setup simplifies onboarding for nontechnical users, such as Okta Identity Governance (OIG) request approvers, who need a ready-made tool without local configuration. See Okta Managed MCP Server documentation.

Reviewer: Add a doc link.

Author: I will add link to okta managed mcp server doc from prod doc later, if I add now, the build will fail due to broken link
